ast-checks/src/missing-patch-comment.rs

```diff
@@ -0,0 +1,106 @@
+use std::collections::HashMap;
+use std::{env, error::Error, fs};
+
+use rnix::types::{List, TypedNode, Wrapper};
+use rnix::SyntaxNode;
+
+use codespan::{FileId, Files};
+
+use serde::{Deserialize, Serialize};
+
+mod comment_finders;
+mod tree_utils;
+
+use comment_finders::{find_comment_above, find_comment_within};
+use tree_utils::walk_keyvalues_filter_key;
```

Comment:
You just need one blank line between `use` and `mod`, if you want to. No reason to have so many visual separators.